### DISCUSSION TITLE
closing a position based on condition

### DISCUSSION THREAD

**Discussion by sfolaxsel** [conor19w/Binance-Futures-Trading-Bot#40] [General]
Is it possible to close a position based on conditions instead of a fixed percentage of the entry price? 
For example, if I want to close the position when the StochRSI indicator enters the overbought zone, how can I achieve this?

**Comment by conor19w** [comment]
Yeah there's no documentation on it but the two heikin ashi strategies use a close_on_condition flag which checks for a condition on the time interval you're trading and then market closes if that position is met. You would have to follow those examples

**Reply by sfolaxsel** [reply]
Thank you! i'll check it!

**Reply by sfolaxsel** [reply]
I'm not sure if it was intentional, but in the process of calculating the Open_H value of Heikin Ashi, you are subtracting 2. 
Changing it to -1 made it consistent with the values provided by the Binance chart. 
Please keep this in mind.

BotClass.py 263 self.Open_H.append((self.Open_H[i - 1] + self.Close_H[i - **2**]) / 2)
